src/Item/Item.router.spec.ts

```diff
@@ -893,7 +894,7 @@ describe('Item router', () => {
                   collection_id: tpCollectionMock.id,
                   eth_address: wallet.address,
                   local_content_hash:
-                    'a630459778465b4882e1cc3e86a019ace033dc06fd2b0d16f4cbab8e075c32f5',
+                    '1590ff765dfe1479b6a6bc46baa493659c57b9b4a99244da7d3ed9464e4c535c',
```

Comment:
Have to recompute the content_hash because the property category is required now.